dadi/lib/model/collections/create.js

```diff
@@ -39,6 +39,25 @@ function create ({
     documents = [documents]
   }
 
+  documents = documents.map(document => {
+    // Add default value for missing fields.
+    Object.keys(this.schema).forEach(field => {
+      if (
+        this.schema[field].default !== undefined &&
+        document[field] === undefined
+      ) {
+        document[field] = this.schema[field].default
+      }
+    })
+
+    // Removing internal API properties from the documents.
+    if (removeInternalProperties) {
+      document = this.removeInternalProperties(document)
+    }
+
+    return document
+  })
+
   // Removing internal API properties from the documents.
   if (removeInternalProperties) {
     documents = documents.map(document => {
```

Comment:
@eduardoboucas looks like this block can go now?

Reply:
Ah, yes! I meant to move it to the `map` but forgot to remove the original. 🤦‍♂️